src/main/java/frc/robot/subsystems/SwerveModule.java

```diff
@@ -40,8 +39,7 @@ public SwerveModule(CANSparkMax driveMotor, CANSparkMax turningMotor, double X,
     m_driveMotor = driveMotor;
     m_turningMotor = turningMotor;
     m_turningEncoder = turningEncoder;
-    m_turningPidController = new PIDController(Constants.SwerveConstants.MODULE_PID_P,
-        Constants.SwerveConstants.MODULE_PID_I, Constants.SwerveConstants.MODULE_PID_D);
+    m_turningPidController = new PIDController(0.3, 0, 0);
```

Comment:
Change the numbers back to constants using just something like SwerveConstants.MODULE_PID_P (Kanav and Lenny)

Reply:
why do you want pid coefficients in constants? it makes tuning a pain, the coefficients are (almost) never used in the same place twice, and it doesn't correspond to any sort of hardware on the bot.